src/ints/int10_modes.cpp

```diff
@@ -1351,6 +1351,8 @@ bool INT10_SetVideoMode(Bit16u mode) {
 		case 7:real_writeb(BIOSMEM_SEG,BIOSMEM_CURRENT_MSR,0x29);break;
 		}
 		break;
+	default:
```

Comment:
Possibly unnecessary, but explicit is always good so I'm not asking for a change here.
 
ISO/IEC 9899:1999, section 6.8.4.2:
[...] If no converted case constant expression matches and there is no default label, no part of the switch body is executed.

Reply:
Yeah, I don't like having defaults in switch statements - when no defaults are used and someone introduces new value to an enum, compiler would warn about all places in code, that needs to be updated…

In this instance, however, original code author decided to remove empty cases… - and coding style in this file encourages using `default:` instead of empty cases. So as a result of this one-liner we go from +4 new warnings to -7 warnings total (because some enum values were missing in here already).

Reply:
That's a worthy cleanup!